account_statement_import_sheet_file/models/account_statement_import_sheet_mapping.py

```diff
@@ -174,6 +174,14 @@ class AccountStatementImportSheetMapping(models.Model):
         help="Set the Header lines number.",
         default="0",
     )
+    skip_empty_lines = fields.Boolean(
+        default=False,
```

Comment:
Maybe `True` is a better default.

Reply:
Yeah, I supposed that, and it's good that you think on that "retro-compatibility", but now on this version that is not so old and established, I was thinking on being more "adventurous" to set a sane default.